src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig

```diff
@@ -58,7 +58,7 @@
             </div>
             <div class="input-field nav-panel-buttom">
                 <ul>
-                    <li class="bold"><a class="waves-effect" href="{{ path('new_entry') }}" id="nav-btn-add"><i class="mdi-content-add"></i></a></li>
+                    <li class="bold"><a class="waves-effect" href="javascript: void(null);" id="nav-btn-add"><i class="mdi-content-add"></i></a></li>
```

Comment:
I think you can let the url inside the href `{{ path('new') }}`. In that case, if every thing goes well, user will have js fully load and will have the js form poped. In any other case, bad network connection, js error, etc ... it will still be able to add an entry because it will be redirected to the `new` form page.